packages/utils/crates/utils-db/src/db.rs

```diff
@@ -118,26 +122,41 @@ impl<T: AsyncKVStorage<Key = Box<[u8]>, Value = Box<[u8]>>> DB<T> {
         db.contains(key.into()).await
     }
 
-    pub async fn get<V: DeserializeOwned>(&self, key: Key) -> Result<V> {
+    // unused and dangerous to use, always use `contains` or `get_or_none`
+    // async fn get<V: DeserializeOwned>(&self, key: Key) -> Result<V> {
+    //     let key: T::Key = key.into();
+    //     let db = self.backend.read().await;
+    //     db.get(key.into()).await.and_then(|v| {
+    //         bincode::deserialize(v.as_ref())
+    //             .map_err(|e| DbError::DeserializationError(format!("during get operation: {}", e.to_string().as_str())))
+    //     })
+    // }
```

Comment:
You can remove this from the codebase completely. It is an incorrect API inherited from the JavaScript, the correct way should be the `get_or_none`.